api_review/personpicture/PersonPictureAPI.md

```diff
@@ -0,0 +1,81 @@
+# Background
+
+Xaml controls are typically implemented using an element tree that’s defined in a template. 
+For example the template for a ComboBox control has a TextBlock in it to display the selected value.
+
+Several controls have properties that are only expected to be useful to these templates, 
+and so there’s a common pattern for the control to expose them in a TemplateSettings property. 
+For example [ComboBox.TemplateSettings](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.ComboBox.TemplateSettings) 
+is used by the ComboBox’s template. See [this doc page](https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/template-settings-classes) for more details. 
+
+This API update introduces this pattern to the PersonPicture control: 
+a new TemplateSettings property and associated PersonPictureTemplateSettings class.
+
+# Description
+
+Provides calculated values that can be referenced as TemplatedParent sources when defining templates 
+for a PersonPicture control. Not intended for general use.
+
+# API Details
+
+```
+namespace Microsoft.UI.Xaml.Controls
+{
+    [webhosthidden]
+    unsealed runtimeclass PersonPicture : Windows.UI.Xaml.Controls.Control
+    {
+        ...
+
+        PersonPictureTemplateSettings TemplateSettings { get; };
+
+        ...
+    }
+
+    [webhosthidden]
+    runtimeclass PersonPictureTemplateSettings : Windows.UI.Xaml.DependencyObject
+    {
+        String EffectiveInitials { get; };

```

Comment:
It's like [EffectiveViewport](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.frameworkelement.effectiveviewportchanged) -- it's the result of combining some other values to provide the "effective" value. But I can understand that it's not a particularly descriptive moniker. I'm open to suggestions. :)

Reply:
Maybe "Calculated"? Also because this is TemplateSettings it maybe is implicitly a "calculated" value so we could also just drop the prefix. But I wanted a different name because it won't necessarily match the PersonPicture.Initials property.

Reply:
The most common word we use for "calculated" is "actual".

Reply:
Changed to Actual.